src/server/pom.xml

```diff
@@ -180,52 +199,59 @@
             <artifactId>dropwizard-shiro</artifactId>
-            <version>0.2.0</version>
+            <version>0.3.0-SNAPSHOT</version>
+            <exclusions>
+                <exclusion>
+                    <groupId>org.apache.shiro</groupId>
+                    <artifactId>shiro-core</artifactId>
+                </exclusion>
+                <exclusion>
+                    <groupId>org.apache.shiro</groupId>
+                    <artifactId>shiro-web</artifactId>
+                </exclusion>
+                <exclusion>
+                    <groupId>org.apache.shiro</groupId>
+                    <artifactId>shiro-jaxrs</artifactId>
+                </exclusion>
+            </exclusions>
+        </dependency>
+        <dependency>
+            <groupId>org.apache.shiro</groupId>
+            <artifactId>shiro-core</artifactId>
+            <version>${shiro-version}</version>
+        </dependency>
+        <dependency>
+            <groupId>org.apache.shiro</groupId>
+            <artifactId>shiro-web</artifactId>
+            <version>${shiro-version}</version>
+        </dependency>
+        <dependency>
+            <groupId>org.apache.shiro</groupId>
+            <artifactId>shiro-jaxrs</artifactId>
+            <version>${shiro-version}</version>
         </dependency>
         <dependency>
             <groupId>io.jsonwebtoken</groupId>
             <artifactId>jjwt</artifactId>
             <version>0.9.1</version>
         </dependency>
         <dependency>
-            <!-- fixes bug around duplicate requests
-              - https://stackoverflow.com/questions/37956741/jersey-resource-receiving-duplicate-requests-from-jersey-client#39377403
-             -->
             <groupId>org.glassfish.jersey.core</groupId>
             <artifactId>jersey-client</artifactId>
-            <!-- can't go up to 2.26 until dropwizard-shiro-0.3.0 is released -->
-            <version>2.25.1</version>
-        </dependency>
-        <dependency>
-            <!-- required to solve dropwizard 1.1.7 & jersery-client 2.25.1 dependency issues -->
-            <groupId>org.glassfish.jersey.core</groupId>
-            <artifactId>jersey-server</artifactId>
-            <version>2.25.1</version>
-        </dependency>
-        <dependency>
-            <!-- required to solve dropwizard 1.1.7 & jersery-client 2.25.1 dependency issues -->
-          <groupId>org.glassfish.jersey.bundles.repackaged</groupId>
-          <artifactId>jersey-guava</artifactId>
-          <version>2.25.1</version>
+            <version>2.33</version>
         </dependency>
         <dependency>
             <groupId>org.glassfish.jersey.media</groupId>
             <artifactId>jersey-media-sse</artifactId>
-            <version>2.25.1</version>
+            <version>2.33</version>
         </dependency>
         <dependency>
             <!-- required to solve dropwizard 1.1.7 & jersery-client 2.25.1 dependency issues -->
```

Comment:
The comment seems outdated, maybe remove it or rephrase it? That said, I did try removing this dependency, and the build failed. So it is definitely still required.

Reply:
right 👍  removed